woocommerce/woocommerce-gutenberg-products-block
# Improve checkout filters arguments description

## What

Improve the description of the arguments passed to the checkout filters.

Fixes #10551 
Fixes #9385

## Why



Currently, the arguments passed to the checkout filters are not documented properly. This makes it challenging for an external developer on how to use these filters correctly. This PR aims to improve the documentation of the arguments passed to the checkout filters.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Look up the *.md files of this PR.
2. Verify that all arguments are documented correctly.
3. Verify that the docs do not contain broken links.
4. Verify that all screenshots contain alt tags.
5. Verify that all footer links _"🐞 Found a mistake, or have a suggestion?"_ link to the corresponding docs.


## Screenshots or screencast

n/a

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog

n/a
